Assignment 6/Ques4.js

```diff
@@ -0,0 +1,37 @@
+// Create a class called `Person` that has properties for `firstName`, `lastName`, and `age`. The class should also have a method called `fullName` that returns the person's full name.
+// Additionally, the class should have a method called `averageAge` that takes in an array of `Person` objects and returns the average age of all the people in the array.
+
+
+class Person{
+    firstName;
+    lastName;
+    age;
+    constructor(firstname,lastname,age)
+    {
+        this.firstName=firstname;
+        this.lastName=lastname;
+        this.age=age;
+    }
+    fullName()
+    {
+        return this.firstName+" "+ this.lastName;
+    }
+    static averageAge(array)
```

Comment:
There are some of the points needs to be consider here, 
- this function is not returning any value 
- upon console logging avgAge this function will return undefined